packages/terra-icon/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Accessibility Guide
```

Comment:
Is this still needed? I don't see any changes made in terra-icon.

Reply:
I can remove this. I figured maybe it helps to know the a11y guide exists now.

Reply:
Removed in a43cfe757670f393f8b39933b55aa689fe1ecb4c